client/webserver/site/src/js/markets.ts

```diff
@@ -458,6 +458,7 @@ export default class MarketsPage extends BasePage {
     // Notification filters.
     app().registerNoteFeeder({
       order: (note: OrderNote) => { this.handleOrderNote(note) },
+      match: () => { this.refreshActiveOrders() },
```

Comment:
This is what fixes it, but you can't do this. This replaces every user order div, which means if the user has details expanded, they will all close on every single update. It's also just way to blunt. You'll need to find the relevant `data-tmpl` element.